src/scss/components/_card-profile.scss

```diff
@@ -62,4 +62,10 @@
     padding: 12px;
     width: 100%;
   }
+
+  &.card--action {
+    .card-profile__highlight {
+      padding-bottom: 57px;
```

Comment:
I know that in most of cases we’ll have this gray box at the bottom of card. But in some cases, the cards on the row, because flexbox, will have a body bigger than another in the right or left side. This cases can make this padding just a security margin between body and footer button.

I don’t know how to solve this corner case, but we can try to discuss this after this patch :)